### ISSUE TITLE
./hg-fast-export.sh: line 177: python2: command not found

### ISSUE THREAD

**Issue by GregWoods** [frej/fast-export#143]
Although it's likely this is a configuration issue, I am stuck

- Windows 10 64 bit
- Python 2.7.15 installed and in the path
- Mercurial 4.7 install, and in the path
- Git 2.18.0.windows.1 installed, and in the path

Running the following command in git-bash:

`./hg-fast-export.sh -r ../pathToHgRepo`

Looking at hg-fast-export.sh, line 177 is a comment, so not sure where to go from here.

Any help appreciated.

**Comment by Troublem4ker** [comment]
I had the same issue. For me a downgrade to tag/v180317 fixed this problem.

**Comment by klm1** [comment]
> I had the same issue. For me a downgrade to tag/v180317 fixed this problem.

Same here -- downgraded version works.

**Comment by AlexP11223** [comment]
Yeah, looks like there is no python2 in Windows Python distributions.

**Comment by klm1** [comment]
Same issue on macOS 10.13.

**Comment by andres-carrilloAdapt2** [comment]
How did you Downgrade?

**Comment by klm1** [comment]
I used:
git checkout tags/v180317

**Comment by AlexP11223** [comment]
You can download it here. https://github.com/frej/fast-export/tree/eca99b61ebb1fa5a2eb2e33a80d05212f5af7539

Or clone and checkout using git.

**Comment by sikmir** [comment]
https://github.com/frej/fast-export/blob/47d330de83055efc5d4eeec6d0593d7aa3640555/hg-fast-export.sh#L29
Just run as:
```bash
$ PYTHON=python ./hg-fast-export.sh -r ../pathToHgRepo
```

**Comment by pdatta1988** [comment]
i could not resolve the issue of python2. If i give PYTHON=python i am getting below error 👎
fast-export/hg-fast-export.py", line 6, in <module>
    from mercurial import node
ImportError: No module named mercurial


Please suggest a solution.

**Comment by AlexP11223** [comment]
> No module named mercurial

it's a different error. You probably don't have Mercurial installed (or it's not in your PATH for some reason).

**Comment by jkfindeisen** [comment]
> Please suggest a solution.

It means you got past that error here and arrived at the next one on Windows. Please make sure you have a Python 2.X with the mercurial module installed running.

**Comment by njackowski** [comment]
Not sure how helpful this is, but I just ran into the same thing.  
Windows 10 x64
Python 2.7.16

After coming across articles on how to install python 2 and 3 side by side, I followed some their recommendations.  In those instances, it is recommended to go into the python install folder and rename python.exe to python2.exe and python3.exe.  In my setup, I copy-pasted "C:\Python27\python.exe" to "C:\Python27\python2.exe" so I have a duplicate.  Conversion worked a treat after that.
